core/string/ustring.h

```diff
@@ -292,6 +292,7 @@ class String {
 	int findn(const char *p_str, int p_from = 0) const; ///< return <0 if failed
 	int rfind(const String &p_str, int p_from = -1) const; ///< return <0 if failed
 	int rfind(const char *p_str, int p_from = -1) const; ///< return <0 if failed
+	int rfind_char(const char32_t &p_char, int p_from = -1) const; ///< return <0 if failed
```

Comment:
char32_t is small enough that passing by value is more efficient.

Reply:
I was hesitant to change this rather than match the existing `find_char` but it's probably small enough to include

Reply:
Looking at the godbolt link you posted on the other pr, it looks like they're both equivalent with optimization flags passed